### PR TITLE
Changelog 3.1

### PR DESCRIPTION
**Pull request by Guaris**
Changelog for 3.1 

Preview: [here](https://deploy-preview-4826--kongdocs.netlify.app/gateway/changelog/)

### CODE REVIEW

File: app/gateway/changelog.md [hunk 1]
```diff
@@ -5,17 +5,372 @@ no_version: true
...
+  - Added referenceable attribute to the `headers` field
+  that could be stored in vaults.
+  [#9611](https://github.com/Kong/kong/pull/9611)
+
+- [**Forward Proxy**](/hub/kong-inc/forward-proxy/) (`forward-proxy`)
+  - `x_headers` field added. This field indicates how the plugin handles the headers
+  `X-Real-IP`, `X-Forwarded-For`, `X-Forwarded-Proto`, `X-Forwarded-Host`, and `X-Forwarded-Port`.
+
+    The field should be set to one of the below and is default to "append":
+    - `append`: append information of this hop to those headers;
```
Comment (cloudjumpercat) [inline app/gateway/changelog.md:128]: The word "hop" seems out of place here

File: app/gateway/changelog.md [hunk 2]
```diff
@@ -5,17 +5,372 @@ no_version: true
...
+    - `transparent`: leave those headers unchanged, as if we were not a proxy;
+    - `delete`: remove all those headers, as if we were the originating client.
+
+    Note that all options respect the trusted IP setting, and will ignore last hop headers if they are not from clients with trusted IPs.
```
Comment (cloudjumpercat) [inline app/gateway/changelog.md:132]:
```suggestion
    Note that all options respect the trusted IP setting, and will ignore last hop-by-hop headers if they are not from clients with trusted IPs.
```

**Review by cloudjumpercat — APPROVED** [review]
Left a small comment about hop headers vs. hop-by-hop headers (I looked up "hop headers" and "hop-by-hop headers" was the only thing that came up, so I wasn't sure if our terminology was correct there)

File: app/gateway/changelog.md (hunk 1 above)
Comment (lena-larionova) [reply app/gateway/changelog.md]: Rephrased based on a bit of reading on hop-by-hop headers.

**Pull request merged**